### ISSUE TITLE
reactnet-webpack template does not include .babelrc file

### ISSUE THREAD

**Issue by bab178** [reactjs/React.NET#1145]
reactnet-webpack template does not include .babelrc file, causing build issues from latest master

**Comment by langebo** [comment]
creating .babelrc containing

``` js
{
  "plugins": [
    "@babel/plugin-proposal-class-properties",
    "babel-plugin-styled-components"
  ],
  "presets": ["@babel/preset-env", "@babel/react"]
}
```

worked for me

**Comment by dustinsoftware** [comment]
Not sure offhand why that’s happening...

**Comment by dustinsoftware** [comment]
Thanks for raising this issue, it was caused by a nuget packaging issue. It's been resolved as of 5.2.8.
